handout/directives/ng_for_directive.md

```diff
@@ -1,6 +1,6 @@
 # NgFor Directive
 
-The `ngFor` directive is a way of repeating a template by using each item of an iterable as that template's context.
+The `ngFor` directive is a way of repeatly adding content inside the ngFor block to the DOM.
```

Comment:
Original wording LGTM too. You can wrap content inside ngFor or use a template tag instead. In either case nothing gets added inside the ngFor. You just end up with multiple copies of the thing you attached the ngFor to.